Instructions/Exercises/01-get-started-azure-openai.md

```diff
@@ -54,76 +61,109 @@ Azure OpenAI provides a web-based portal named **Azure OpenAI Studio**, that you
 
 ## Use the Chat playground
 
-The *Chat* playground provides a chatbot interface for GPT 3.5 and higher models.
+Now that you've deployed a model, you can use it to generate responses based on natural language prompts. The *Chat* playground in Azure OpenAI Studio provides a chatbot interface for GPT 3.5 and higher models.
 
 > **Note:** The *Chat* playground uses the *ChatCompletions* API rather than the older *Completions* API that is used by the *Completions* playground. The Completions playground is provided for compatibility with older models.
 
-1. In the **Playground** section, select the **Chat** page. The **Chat** playground page consists of three main sections:
+1. In the **Playground** section, select the **Chat** page. The **Chat** playground page consists of three main panels (which may be arranged right-to-left horizontally, or top-to-bottom vertically depending on your screen resolution):
     - **Assistant setup** - used to set the context for the model's responses.
     - **Chat session** - used to submit chat messages and view responses.
     - **Configuration** - used to configure settings for the model deployment.
-1. In the **Configuration** section, ensure that your model deployment is selected.
-1. In the **Assistant setup** section, in the **System message** box, replace the current text with the following statement: `The system is an AI teacher that helps people learn about AI`.
+1. In the **Configuration** panel, ensure that your gpt-35-turbo-16k model deployment is selected.
+1. In the **Assistant setup** panel, review the default **System message**, which should be *You are an AI assistant that helps people find information.* The system message is included in prompts submitted to the model, and provides context for the model's responses; setting expectations about how an AI agent based on the model should interact with the user.
```

Comment:
It's now called just "Setup" - I changed it a couple places but didn't get this one